public/app/features/alerting/unified/initAlerting.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { config } from '@grafana/runtime';
+
+import { addCustomRightAction } from '../../dashboard/components/DashNav/DashNav';
+
+import { AlertRulesToolbarButton } from './integration/AlertRulesDrawer';
+
+export function initAlerting() {
+  addCustomRightAction({
+    show: () => config.unifiedAlertingEnabled,
+    component: ({ dashboard }) => dashboard && <AlertRulesToolbarButton dashboardUid={dashboard.uid} />,
+    index: -2,
+  });
+}
```

Comment:
if we use React.lazy here for the AlertRulesToolbarButton we would not include all of alerting into the app webpack chunk. 

@dprokop or am I missing something?

Reply:
I added React.lazy here to delay the toolbar button loading and also I lazily load the Alert rules drawer so the toolbar button footprint is really small